tests/author.test.js

```diff
@@ -0,0 +1,162 @@
+const app = require("../src/app");
+const { expect } = require("chai");
+const request = require("supertest");
+const { Author } = require("../src/models");
+const { beforeEach } = require("mocha");
+const faker = require("faker");
+
+describe("/authors", () => {
+  before(async () => Author.sequelize.sync());
+
+  beforeEach(async () => {
+    await Author.destroy({ where: {} });
+  });
+
+  describe("with no records in the database", () => {
+    describe("POST /authors", () => {
+      it("Should create a new author in the database", async () => {
+        const authorData = {
+          author: faker.name.findName(),
+        };
+        const response = await request(app).post("/authors").send(authorData);
+
+        expect(response.status).to.equal(201);
+
+        const newAuthorRecord = await Author.findByPk(response.body.id, {
+          raw: true,
+        });
+
+        expect(newAuthorRecord.author).to.equal(authorData.author);
+      });
+
+      it("Should return 500 if author is empty", async () => {
+        const authorData = {
+          author: "",
+        };
+
+        const result = await request(app).post("/authors").send(authorData);
```

Comment:
Small format suggestion:
```suggestion
        const result = await request(app).post("/authors").send(authorData);

```